kubeflow/fairing/kubernetes/manager.py

```diff
@@ -217,7 +217,7 @@ def secret_exists(self, name, namespace):
         :returns: bool: True if the secret exists, otherwise return False.
 
         """
-        secrets = client.CoreV1Api().list_namespaced_secret(namespace)
+        secrets = client.CoreV1Api().read_namespaced_secret(namespace)
```

Comment:
The `read_namespaced_secret` function should be like this `read_namespaced_secret(name, namespace, pretty=pretty, exact=exact, export=export)` from docs https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CoreV1Api.md#read_namespaced_secret. The `name` should be required args.